content/_code-samples/trade-in-the-decentralized-exchange/js/README.md

```diff
@@ -0,0 +1,5 @@
+# Trade in the Decentralized Exchange
+
+This code demonstrates how to buy a (fungible) token on the XRP Ledger's decentralized exchange (DEX). For a detailed explanation, see <https://xrpl.org/trade-in-the-decentralized-exchange.html>.
```

Comment:
I think a bit more context on what the detailed explanation is pointing to could be helpful.

```suggestion
This code demonstrates how to buy a (fungible) token on the XRP Ledger's decentralized exchange (DEX). For a detailed explanation of how to trade using the DEX, see <https://xrpl.org/trade-in-the-decentralized-exchange.html>.
```